src/Microsoft.DotNet.Interactive.FSharp/FSharpKernel.fs

```diff
@@ -412,16 +412,16 @@ type FSharpKernel () as this =
     member _.PackageRestoreContext = _packageRestoreContext.Value
 
     interface IKernelCommandHandler<RequestCompletions> with
-        member this.HandleAsync(command: RequestCompletions, context: KernelInvocationContext) = handleRequestCompletions command context |> Async.StartAsTask :> Task
+        member this.HandleAsync(command: RequestCompletions, context: KernelInvocationContext) = handleRequestCompletions command context
 
     interface IKernelCommandHandler<RequestDiagnostics> with
-        member this.HandleAsync(command: RequestDiagnostics, context: KernelInvocationContext) = handleRequestDiagnostics command context |> Async.StartAsTask :> Task
+        member this.HandleAsync(command: RequestDiagnostics, context: KernelInvocationContext) = handleRequestDiagnostics command context 
 
     interface IKernelCommandHandler<RequestHoverText> with
         member this.HandleAsync(command: RequestHoverText, context: KernelInvocationContext) = handleRequestHoverText command context |> Async.StartAsTask :> Task
 
     interface IKernelCommandHandler<SubmitCode> with
-        member this.HandleAsync(command: SubmitCode, context: KernelInvocationContext) = handleSubmitCode command context |> Async.StartAsTask :> Task
+        member this.HandleAsync(command: SubmitCode, context: KernelInvocationContext) = handleSubmitCode command context
 
     interface IKernelCommandHandler<ChangeWorkingDirectory> with
         member this.HandleAsync(command: ChangeWorkingDirectory, context: KernelInvocationContext) = handleChangeWorkingDirectory command context |> Async.StartAsTask :> Task
```

Comment:
should we complete the migration to task ?

Reply:
Probably. At this point I'm stabbing to figure out why one of a few different random tests keep failing in CI.